packages/http/src/resolvers/http-and-file.ts

```diff
@@ -22,8 +22,8 @@ export const httpAndFileResolver = new Resolver({
     },
   },
 
-  parseResolveResult: async opts => {
+  parseResolveResult: opts => {
```

Comment:
would we not want async and await instead of the Promise.resolve() ?

Reply:
Normally yes; the thing is this is a false async function, so there's never going to be an await; this is the reason ESLint is complaining.

I'm ok removing the rule if bothers the team though, I'm not opinionated on this one.